lib/fhir/mongoid/models.rb

```diff
@@ -7,3 +7,7 @@
 require_relative '../../../app/models/cqm/cql_statement_dependency'
 require_relative '../../../app/models/cqm/population_set'
 require_relative '../../../app/models/cqm/data_element'
+
+# Require all files in directories under lib/ext that contain custom extensions
+# on the autogenerated classes
+Dir[File.join(__dir__, '../../ext/**', '*.rb')].each{|file| require_relative file}
```

Comment:
Codacy found an issue: [Space between { and | missing.](https://app.codacy.com/gh/MeasureAuthoringTool/fhir-mongoid-models/pullRequest?prid=5999190)